solr/core/src/java/org/apache/solr/search/HashQParserPlugin.java

```diff
@@ -114,7 +116,12 @@ public LongValues getValues(LeafReaderContext ctx, DoubleValues scores) throws I
             @Override
             public long longValue() throws IOException {
               //TODO: maybe cache hashCode if same ord as prev doc to save lookupOrd?
-              return atDoc ? values.lookupOrd(values.ordValue()).hashCode() : 0;
+              return atDoc ? hashCode(values.lookupOrd(values.ordValue())) : 0;
+            }
+
+            private long hashCode(BytesRef bytesRef) {
```

Comment:
```suggestion
            private static int hashCode(BytesRef bytesRef) {
```

or

```suggestion
            private static long hashCode(BytesRef bytesRef) {
```

Reply:
anonymous inner classes can't have static methods.  But I'll change to int from long